src/App.vue

```diff
@@ -630,8 +630,8 @@ export default class App extends Mixins(
         throw error // go to catch()
       }
 
-      // load parties only for SP/GP
-      this.isTypeFirm && this.loadPartiesInformation()
+      // load parties only for SP/GP businesses
+      if (this.isTypeFirm && this.getBusinessId) this.loadPartiesInformation()
```

Comment:
Only existing businesses can load parties, and only existing businesses have a business ID.

(IAs and Registrations have a temp reg number instead, and cannot load parties.)